sdk/avs/Microsoft.Azure.Management.Avs/src/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure VMware Solution Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure VMware Solution.")]
 
-[assembly: AssemblyVersion("2.0.0")]
-[assembly: AssemblyFileVersion("2.0.0.1")]
+[assembly: AssemblyVersion("3.0.0")]
+[assembly: AssemblyFileVersion("3.0.0.1")]
```

Comment:
I think you can change the `AssemblyFileVersoin` to `3.0.0.0`

Reply:
I'll leave as `3.0.0`. Sounds like that will work.